lib/open_mail_app.dart

```diff
@@ -13,6 +13,7 @@ class OpenMailApp {
   OpenMailApp._();
 
   static const MethodChannel _channel = const MethodChannel('open_mail_app');
+  static List<String> _filterList = <String>[];
```

Comment:
Should we set any default? Maybe include PayPal since others will run into it as well?

Reply:
We certainly can, though for open source, I would generally try to err on the side of less bias rather than more. However, the fact that we didn't catch the PayPal issue for so long means that others might not notice it either so... given that the intention of the app is to provide a simple clean way for finding and launching *mail* apps, seems reasonable.

Reply:
Went with it. Will also add some reference to the default filter list in the dartdoc.